pkg/chain/ethereum/ethutil/ethutil.go

```diff
@@ -21,6 +21,15 @@ import (
 
 var logger = log.Logger("keep-ethutil")
 
+// EthereumClient wraps the core `bind.ContractBackend` interface with
+// some other interfaces allowing to expose additional methods provided
+// by client implementations.
+type EthereumClient interface {
+	bind.ContractBackend
+	ethereum.ChainReader
+	ethereum.TransactionReader
```

Comment:
Do we use `TransactionByHash` or `TransactionReceipt` from `ethereum.TransactionReader` anywhere?

Reply:
We use `TransactionReceipt` in the `mine_waiter.go`